dockers/docker-database/database_config.json.j2

```diff
@@ -72,6 +72,11 @@
             "id" : 11,
             "separator": "|",
             "instance" : "redis"
+        },
+        "GLOBAL_STATE_DB" : {
```

Comment:
now, bring up chassis-db service [PR5283](https://github.com/Azure/sonic-buildimage/pull/5283) merged into master, can you rebase your branch so that you will see both CHASSIS_APP_DB and redis_chassis server instance are already defined in this file.

Reply:
We are adding a CHASSIS_STATE_DB so that the pmon state is kept separate from the data-path state in CHASSIS_APP_DB. This is inline with what happens on local redis instance.